internal/plugins/manifests/v2/init.go

```diff
@@ -181,6 +181,11 @@ endif
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
 OPERATOR_SDK_VERSION ?= %[3]s
+
+# Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
```

Comment:
Does LOCALBIN already exist in the makefile that we are going to be adding this information to?

Reply:
Good catch! I'll update.